src/js/Controllers/UIController.js

```diff
@@ -196,7 +196,10 @@ class UIController {
                     rpc.params.buttonName,
                     true
                 ))
-                this.listener.send(RpcFactory.SubscribeButton(rpc.params.appID, rpc.params.buttonName));
+                if(!rpc.params.buttonName || !rpc.params.appID) {
```

Comment:
@LiliiaShlikhtLuxoft Are you trying to check boolean params? `buttonName` and `appID` is a boolean ?

Reply:
@ValeriiMalkov this condition has been created according to generic HMI style. I see through all files the same structure. Should I change it?